packages/arb-token-bridge-ui/src/util/orbitChainsList.ts

```diff
@@ -206,6 +206,49 @@ export const orbitMainnets: {
       }
     }
   },
+  70701: {
+    chainId: 70701,
+    confirmPeriodBlocks: 43200,
+    ethBridge: {
+      bridge: '0x53D82686BC9827fEc03bcEe661B37b855A18EcA9',
+      inbox: '0xEe30EfcaF812d10e1EFE25E9458f76a39DAD3239',
+      outbox: '0xa4270256B160C3Ebec2d6914a906c7EC38D8d072',
+      rollup: '0x330F8fEB25f3427cABA32446728C36ae67f2135b',
+      sequencerInbox: '0x6eE94AD8057Fd7Ba4d47bb6278a261c8a9FD4E3f'
+    },
+    explorerUrl: 'https://explorer-proofofplay-boss-mainnet.t.conduit.xyz',
+    rpcUrl: 'https://rpc.boss.proofofplay.com',
+    name: 'Proof of Play Boss',
+    slug: 'pop-boss',
+    parentChainId: 42161,
+    isCustom: true,
+    isTestnet: false,
+    tokenBridge: {
+      parentCustomGateway: '0x7D25Cc5b71AC932F696F3f4Da109e867395f4F05',
+      parentErc20Gateway: '0x298eb8d9f2F046AC60c01535fad40320CCdeB7c0',
+      parentGatewayRouter: '0x48e067CC8eedc2f6E6dfB7192E7E32158A7de0E2',
+      parentMultiCall: '0x90B02D9F861017844F30dFbdF725b6aa84E63822',
+      parentProxyAdmin: '0x490C4c92Ea9FF02EE8277222C66afD80Bfb1b7c1',
+      parentWeth: '0x82aF49447D8a07e3bd95BD0d56f35241523fBab1',
+      parentWethGateway: '0x7480A89d1e360e512493489711EC7bc8eE7acfD6',
+      childCustomGateway: '0xA534bFDaBBac7F0511E97bDFCE8aB72bd7380355',
+      childErc20Gateway: '0x298eb8d9f2F046AC60c01535fad40320CCdeB7c0',
```

Comment:
https://explorer-proofofplay-boss-mainnet.t.conduit.xyz/address/0x298eb8d9f2F046AC60c01535fad40320CCdeB7c0

on their explorer it says this address is EOA
and this address they gave us appears to be the same as `parentErc20Gateway`

are we sure this is not a mistake?

Reply:
Thanks, yes ERC20 gateway is the standard gateway. We will clarify it on the form to prevent future mistakes, thanks!